workflow/rules/post-analysis.smk

```diff
@@ -0,0 +1,181 @@
+    input:
+        "resources/ref/genome.bed",
+        expand("results/big_wig/{sample}.bigWig", sample=samples.index)
+    output:
+        "results/IGV/merged_library.bigWig.igv.txt"
+    shell:
+        "find {input} -type f -name '*.bigWig' -exec echo -e 'results/big_wig/\"{{}}\"\t0,0,178' \;  > {output}"
+
+rule compute_matrix:
+    input:
+         bed="resources/ref/genome.bed",
+         bigwig=expand("results/big_wig/{sample}.bigWig", sample=samples.index)
+    output:
+        # Usable output variables, their extensions and which option they implicitly call are listed here:
+        #         https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/deeptools/computematrix.html.
+        matrix_gz="results/deeptools/matrix_files/matrix.gz",
+        matrix_tab="results/deeptools/matrix_files/matrix.tab"
+    log:
+        "logs/deeptools/compute_matrix.log"
+    params:
+        command="scale-regions",
+        extra="--regionBodyLength 1000 "
+              "--beforeRegionStartLength 3000 "
+              "--afterRegionStartLength 3000 "
+              "--skipZeros "
+              "--smartLabels "
+              "--numberOfProcessors 2 "
+    wrapper:
+        "0.64.0/bio/deeptools/computematrix"
+
+rule plot_profile:
+    input:
+         "results/deeptools/matrix_files/matrix.gz"
+    output:
+        # Usable output variables, their extensions and which option they implicitly call are listed here:
+        #         https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/deeptools/plotprofile.html.
+        plot_img="results/deeptools/plot_profile.pdf",
+        data="results/deeptools/plot_profile_data.tab"
+    log:
+        "logs/deeptools/plot_profile.log"
+    params:
+        ""
+    wrapper:
+        "0.64.0/bio/deeptools/plotprofile"
+
+rule plot_heatmap:
+    input:
+         "results/deeptools/matrix_files/matrix.gz"
+    output:
+        # Usable output variables, their extensions and which option they implicitly call are listed here:
+        #         https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/deeptools/plotheatmap.html.
+        heatmap_img="results/deeptools/heatmap.pdf",
+        heatmap_matrix="results/deeptools/heatmap_matrix.tab"
+    log:
+        "logs/deeptools/heatmap.log"
+    params:
+        ""
+    wrapper:
+        "0.64.0/bio/deeptools/plotheatmap"
+
+rule phantompeakqualtools:
+    input:
+         "results/orphan_rm_sorted/{sample}.bam"
+    output:
+        res_phantom="results/phantompeakqualtools/{sample}.phantompeak.spp.out",
+        r_data="results/phantompeakqualtools/{sample}.phantompeak.Rdata",
+        plot="results/phantompeakqualtools/{sample}.phantompeak.pdf"
+    threads:
+        8
+    conda:
+        "../envs/phantompeakqualtools.yaml"
+    shell:
+        "Rscript -e \"library(caTools); source('../workflow/scripts/run_spp.R')\" "
+        "-c={input} -savp={output.plot} -savd={output.r_data} "
+        "-out={output.res_phantom} -p={threads}"
```

Comment:
I'd add logging files to these phantompeakqualtools rules, to capture anything that the `Rscript` run might output. This should work with by specifying a `log:` entry above and then wrapping the shell command as follows:
```suggestion
        "( Rscript -e \"library(caTools); source('../workflow/scripts/run_spp.R')\" "
        "  -c={input} -savp={output.plot} -savd={output.r_data} "
        "  -out={output.res_phantom} -p={threads} 2>&1 ) >{log}"
```